src/test/java/com/ironcorelabs/tenantsecurity/kms/v1/DevIntegrationTest.java

```diff
@@ -56,7 +56,7 @@ private void assertEqualBytes(byte[] one, byte[] two) throws Exception {
     }
 
     private CompletableFuture<TenantSecurityKMSClient> getClient() {
-        return TenantSecurityKMSClient.create("http://localhost:7777", this.INTEGRATION_API_KEY);
+        return TenantSecurityKMSClient.create(TestSettings.TSP_ADDRESS + TestSettings.TSP_PORT, this.INTEGRATION_API_KEY);
```

Comment:
Where is this coming from? Did you forget to add a file to this commit?

Reply:
Most def.

Reply:
Should be up now.